Habitica/res/values-de/strings.xml

```diff
@@ -790,15 +790,27 @@ Die Quest-Schriftrolle wird an den Quest-Besitzer zurückgegeben.</string>
   <string name="setup_task_reward_notes">Schau Fernsehen, spiel ein Spiel, iss etwas Süßes, es liegt ganz an Dir!</string>
   <string name="visit_website">Internetseite besuchen</string>
   <string name="gift_title">Wen würdest Du gerne beschenken?</string>
-<string name="expand_notes">Mehr lesen</string>
-    <string name="collapse_notes">Weniger anzeigen</string>
-    <string name="last_login">Letzte Anmeldung</string>
-    <string name="total_checkins">Gesamtanmeldungen</string>
-    <string name="two_handed">Zweihändig</string>
-    <string name="confirmUsername">Benutzernamen bestätigen</string>
-    <string name="username_level">\@%s ・Lvl %d</string>
-    <string name="verification_pet">Eines dieser Veteranen-Haustiere wartet auf Dich wenn Du die Bestätigung abgeschlossen hast!</string>
-    <string name="gift_subscription">Abonnement verschenken</string>
-    <string name="create_party">Gruppe erstellen</string>
-
+  <string name="expand_notes">Mehr lesen</string>
+  <string name="collapse_notes">Weniger anzeigen</string>
+  <string name="last_login">Letzte Anmeldung</string>
+  <string name="total_checkins">Gesamtanmeldungen</string>
+  <string name="two_handed">Zweihändig</string>
+  <string name="confirmUsername">Benutzernamen bestätigen</string>
+  <string name="username_level">\@%s ・Lvl %d</string>
+  <string name="verification_pet">Eines dieser Veteranen-Haustiere wartet auf Dich wenn Du die Bestätigung abgeschlossen hast!</string>
+  <string name="gift_subscription">Abonnement verschenken</string>
+  <string name="create_party">Gruppe erstellen</string>
+  <string name="notifications">Mitteilungen</string>
+  <string name="no_notifications_title">Du bist auf dem Laufenden!</string>
+  <string name="no_notifications_text">Die Mitteilungsfeen geben dir eine heftige Runde Applaus! Gut gemacht!</string>
+  <string name="dismiss_all">Alle entfernen</string>
+  <string name="new_bailey_update">Neues Update von Bailey!</string>
+  <string name="new_msg_guild"><![CDATA[<b>%1$s</b> hat neue Beiträge]]></string>
```

Comment:
Have you tested this? I'm not sure if the `<![CDATA[` part works?

Reply:
Nevermind. Just noticed that this is also in the english versions and also in the ones already merged in and it works.